packages-exp/auth-compat-exp/src/auth.ts

```diff
@@ -199,28 +200,29 @@ export class Auth
     );
   }
   async setPersistence(persistence: string): Promise<void> {
-    function convertPersistence(
-      auth: exp.Auth,
-      persistenceCompat: string
-    ): exp.Persistence {
-      _validatePersistenceArgument(auth, persistence);
-      switch (persistenceCompat) {
-        case Persistence.SESSION:
-          return exp.browserSessionPersistence;
-        case Persistence.LOCAL:
-          return isIndexedDBAvailable()
-            ? exp.indexedDBLocalPersistence
-            : exp.browserLocalPersistence;
-        case Persistence.NONE:
-          return exp.inMemoryPersistence;
-        default:
-          return exp._fail(exp.AuthErrorCode.ARGUMENT_ERROR, {
-            appName: auth.name
-          });
-      }
+    _validatePersistenceArgument(this.auth, persistence);
+    let converted;
+    switch (persistence) {
```

Comment:
Based on my testing, legacy SDK only accepts `local`, `session`, or `none` as lowercase, not UPPERCASE. Did we unintentionally introduce a breaking change here?

Reply:
No we should fix that :P

Reply:
Changed compat to lowercase while leaving `auth-exp` unchanged for now. This will probably also help us tell if we mistaken one for another.